cli/src/main/java/com/box/l10n/mojito/cli/command/CommitCreateCommand.java

```diff
@@ -151,7 +152,8 @@ CommitInfo readFromGit() {
               revCommit.getName(),
               authorIdent.getEmailAddress(),
               authorIdent.getName(),
-              Instant.ofEpochSecond(revCommit.getCommitTime()).toDateTime());
+              ZonedDateTime.ofInstant(
```

Comment:
similar but maybe shorter / closer to original code: `Instant.ofEpochSecond(revCommit.getCommitTime()).atZone(systemDefault())`